CHANGELOG.adoc

```diff
@@ -5,6 +5,56 @@ https://spring.io/projects/spring-cloud-gcp[Spring Cloud GCP] is a set of integr
 This document provides a high-level overview of the changes introduced in Spring Cloud GCP by release.
 For a detailed view of what has changed, refer to the https://github.com/spring-cloud/spring-cloud-gcp/commits/master[commit history] on GitHub.
 
+== 1.2.3.RELEASE (2020-05-29)
+
+=== General
+
+* Upgrade to latest libraries bom and sql socket factory (https://github.com/spring-cloud/spring-cloud-gcp/issues/2373[#2373])
+* Make transaction managers conditional on enabled flag for Spanner and Datastore (https://github.com/spring-cloud/spring-cloud-gcp/issues/2376[#2376])
+
+=== Secret Manager 
+
```

Comment:
I would add a more explicit statement that there are breaking changes between 1.2.2 and 1.2.3 for secret manager.

Maybe somehting like:

"Version 1.2.3.RELEASE introduces breaking changes to using Secret Manager as a Spring Boot property source. Please see the [reference documentation](https://cloud.spring.io/spring-cloud-static/spring-cloud-gcp/1.2.3.RELEASE/reference/html/#secret-manager-property-source) for the updated syntax for how to specify secrets in your application's `.properties` files."

Reply:
Done.